software/Makefile

```diff
@@ -16,17 +16,19 @@ SED ?= sed
 # 120 MB - 4096
 TOP_OF_MEM_RISC_V=0x877FF000
 
-FILTER_OUT = $(foreach v,$(2),$(if $(findstring $(1),$(v)),,$(v)))
+# test command avoids error if $(SDK_DIR)/prog does not yet exist
+all-riscv-binaries:=$(shell test -d $(SDK_DIR)/prog && find $(SDK_DIR)/prog -iname "*.riscv")
```

Comment:
I would do:

ifeq ($(SDK_DIR),)
$error(“Please set…”)

And keep the target the same

Reply:
the test is to check that sdk/prog exists, not SDK_DIR. sdk/prog is where the built binaries live in the SDK, so that is where we should search for them from ZP